llvm/lib/Analysis/InlineCost.cpp

```diff
@@ -1153,6 +1154,7 @@ class InlineCostFeaturesAnalyzer final : public CallAnalyzer {
   // heuristics in the ML inliner.
   static constexpr int JTCostMultiplier = 4;
   static constexpr int CaseClusterCostMultiplier = 2;
+  static constexpr int SwitchDefaultDestCostMultiplier = 2;
```

Comment:
Defined this constant but hardcoded 2 above?

Reply:
I just copied it from `CaseClusterCostMultiplier`.